### DISCUSSION TITLE
After pause, the print make a purge tower

### DISCUSSION THREAD

**Discussion by pollosud** [OrcaSlicer/OrcaSlicer#4960] [Ideas]
Hello, I want to put the printer in puse, then, when I restart, i want to do a tower purge.Is it possible?

**Comment by pollosud** [comment]
I want to do the purge tower when I restart the print

Il giorno mer 10 apr 2024 alle ore 21:04 vgdh ***@***.***> ha
scritto:

**Comment by MarioL77** [comment]
If you have a klipper then EVERYTHING is possible - you need a macro for that
:-)
